src/Microsoft.OData.Core/JsonLight/ODataJsonLightWriterUtils.cs

```diff
@@ -54,5 +55,47 @@ internal static void WriteInstanceAnnotationName(this IJsonWriter jsonWriter, st
 
             jsonWriter.WriteName(JsonLightConstants.ODataPropertyAnnotationSeparatorChar + annotationName);
         }
+
+        /// <summary>
+        /// Writes the 'value' property name asynchronously.
+        /// </summary>
+        /// <param name="jsonWriter">The JSON writer to write to.</param>
+        internal static async Task WriteValuePropertyNameAsync(this IJsonWriterAsync jsonWriter)
+        {
+            Debug.Assert(jsonWriter != null, "jsonWriter != null");
+
+            await jsonWriter.WriteNameAsync(JsonLightConstants.ODataValuePropertyName)

```

Comment:
>await [](start = 12, length = 5)

Note: we can "elide" the await for pass-through calls and just return the Task. See, for example, discussion at [https://blog.stephencleary.com/2016/12/eliding-async-await.html](https://blog.stephencleary.com/2016/12/eliding-async-await.html)